src/test/java/seedu/wife/testutil/IncreaseFoodDescriptorBuilder.java

```diff
@@ -0,0 +1,41 @@
+package seedu.wife.testutil;
+
+import seedu.wife.logic.commands.foodcommands.IncreaseCommand.IncreaseFoodDescriptor;
+import seedu.wife.model.food.Food;
+import seedu.wife.model.food.Quantity;
+
+/**
+ * A utility class to help with building IncreasePersonDescriptor objects.
```

Comment:
Can you change this to `IncreaseFoodDescriptor`? Same thing for line 31 as well

Reply:
resolved

Reply:
resolved